assets/js/amp-editor-blocks.js

```diff
@@ -503,13 +499,12 @@ var ampEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 				el( TextControl, {
 					label: __( 'Height' ),
 					value: height,
-					type: 'number',
 					min: 1,
 					onChange: function( nextHeight ) {
 						props.setAttributes( { height: nextHeight } );
 					}
 				} ),
-				maxFont > height && el(
+				parseInt( maxFont ) > parseInt( height ) && el(
```

Comment:
I think for easier reading and to be DRY, we could `parseInt()` these variables just inside the preceding `if ( ) {` statement. Also, it's good practice to explicitly state the 2nd parameter for `parseInt()` which in this case would be 10 like `parseInt( maxFont, 10 )` .

Reply:
Thanks, makes sense!